joblib/testing.py

```diff
@@ -8,25 +8,14 @@
 import re
 import subprocess
 import threading
-import unittest
 
 import pytest
 import _pytest
 
 from joblib._compat import PY3_OR_LATER
 
 
-_dummy = unittest.TestCase('__init__')
-
-assert_raises = _dummy.assertRaises
-pytest_assert_raises = pytest.raises
-
-try:
-    assert_raises_regex = _dummy.assertRaisesRegex
-except AttributeError:
-    # Python 2.7
-    assert_raises_regex = _dummy.assertRaisesRegexp
-
+assert_raises = pytest.raises
```

Comment:
Done in next commit. Although I am skeptical about the changing of old `assert_raises` calls. Either we should avoid this renaming or replce old `assert_raises` calls with `with` blocks as well...